### ISSUE TITLE
[Suggestion] SM64JS Update Schedule

### ISSUE THREAD

**Issue by ghost** [sm64js/sm64js#686]
I have a suggestion for the update schedule for sm64js: Maybe you could do a big update at the end of every month. IE: Do a big update on May 31. Also maybe instead of just doing some improvements and tiny updates every once in a while, do them every Friday or Saturday.

**Comment by ImaginaryNeon** [comment]
I don't think that they can really do this right now because (to my knowledge) they are kind of short staffed atm, and it's probably hard (and stressful) to add big updates every month.
(Although I don't know exactly how hard adding stuff is, since I have only really coded stuff in [Khan Academy](https://www.khanacademy.org/profile/ImaginaryNeon), which is likely easier to edit than javascript or whatever they're using to make this. (And even then, I usually just add stuff to preexisting games that I like.)) 
Also keep in mind that at one point, they'd just run out of stuff to add.

**Comment by ghost** [comment]
hm good point. ig eventually they would run out of stuff to add. The reason i made this issuse is because i never know when they are going to add something new and i just wanted them to have a little schedule instead of just randomly adding stuff every once in a while.

**Comment by AgentXLP** [comment]
I'm the main dev for everything and I have other side projects to work on as well, I update the game when I can but an update schedule to have at least is a good idea.

**Comment by MatthewMario64** [comment]
If we get a schedule, that would be amazing and maybe we could add ideas for updates! I keep thinking about when you guys would add other Mario maps from other games.

**Comment by abnormalhare** [comment]
I've talked to X about doing stuff like this. I only work on the Vanilla side, but it'd be great to have something like a "snapshot" system.
